api/services/account_service.py

```diff
@@ -505,16 +505,19 @@ def create_tenant(name: str, is_setup: Optional[bool] = False) -> Tenant:
     def create_owner_tenant_if_not_exist(
         account: Account, name: Optional[str] = None, is_setup: Optional[bool] = False
     ):
-        """Create owner tenant if not exist"""
```

Comment:
I don't think this should be removed.

Reply:
> I don't think this should be removed.

Thank you for your feedback, @crazywoola . Just to clarify, I’m not suggesting the remove it. Instead, I propose moving the functionality that checks if a user has a workspace to occur before the is_allow_create_workspace validation. If the user already has a workspace, we should bypass the check for whether they can create a new one. This change aims to streamline the login process and prevent unnecessary login failures for users with existing workspaces. I appreciate your insights and am open to further discussion on this approach.

Reply:
I may have forgotten some of this code, maybe @ZhouhaoJiang can help you with more insights.